bundle/config/mutator/python/python_logger.go

```diff
@@ -0,0 +1,93 @@
+package python
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"strings"
+
+	"github.com/databricks/cli/libs/log"
+)
+
+type stderrLogger struct {
+	ctx context.Context
+	buf bytes.Buffer
+}
+
+type level string
+
+const levelError level = "ERROR"
+const levelInfo level = "INFO"
+const levelDebug level = "DEBUG"
+const levelWarning level = "WARNING"
+
+type logEntry struct {
+	Level   level  `json:"level,omitempty"`
+	Message string `json:"message"`
+}
```

Comment:
Is this a standard format with Python logging?

Reply:
There is no standard. Logger implementation looks like this:
```
class JsonFormatter(logging.Formatter):
    def format(self, record):
        log_record = {
            'level': record.levelname,
            'message': record.getMessage()
        }

        return json.dumps(log_record)
```

Perhaps, we can use go format instead, since it has some standard.

Reply:
We almost use the Golang format, with the difference that "level" shouldn't be capitalized. Unfortunately, there is no built-in struct for it. I can also see that we will eventually want to add custom fields, for instance, to log traceback. So likely we would need our own custom format.

Reply:
I will remove it now, and add as a follow-up